c10/test/util/complex_test_common.h

```diff
@@ -389,6 +389,32 @@ MAYBE_GLOBAL void test_arithmetic() {
   test_arithmetic_<double>();
 }
 
+template<typename T, typename int_t>
+static void test_binary_ops_for_int_type_(T real, T img, int_t num) {
```

Comment:
`C10_HOST_DEVICE `

Reply:
Added

Reply:
I had to remove this keyword, otherwise I keep getting error on Windows CI:

```
C:/Users/circleci/project\c10/test/util/complex_test_common.h(407): error: calling a __host__ function from a __host__ __device__ function is not allowed
          detected during instantiation of "void arithmetic::test_binary_ops_for_all_int_types_(T, T, int8_t) [with T=float]" 
(414): here

C:/Users/circleci/project\c10/test/util/complex_test_common.h(408): error: calling a __host__ function from a __host__ __device__ function is not allowed
          detected during instantiation of "void arithmetic::test_binary_ops_for_all_int_types_(T, T, int8_t) [with T=float]" 
(414): here

C:/Users/circleci/project\c10/test/util/complex_test_common.h(409): error: calling a __host__ function from a __host__ __device__ function is not allowed
          detected during instantiation of "void arithmetic::test_binary_ops_for_all_int_types_(T, T, int8_t) [with T=float]" 
(414): here

C:/Users/circleci/project\c10/test/util/complex_test_common.h(410): error: calling a __host__ function from a __host__ __device__ function is not allowed
          detected during instantiation of "void arithmetic::test_binary_ops_for_all_int_types_(T, T, int8_t) [with T=float]" 
(414): here
```